client/my-sites/domains/domain-management/components/inbound-transfer-verification/index.jsx

```diff
@@ -0,0 +1,50 @@
+/**
+ * External dependencies
+ *
+ */
+import PropTypes from 'prop-types';
+import React from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import EmailVerificationCard from 'my-sites/domains/domain-management/components/email-verification';
+import { resendInboundTransferEmail } from 'lib/domains';
+
+class InboundTransferEmailVerificationCard extends React.Component {
+	static propTypes = {
+		selectedDomainName: PropTypes.string.isRequired,
+		selectedSiteSlug: PropTypes.string.isRequired,
+	};
+
+	render() {
+		return (
+			<div>
+				<EmailVerificationCard
+					verificationExplanation={ this.props.translate(
+						'We need to check your contact information to make sure you can be reached. Please verify your ' +
```

Comment:
Hi! I've found a possible matching string that has already been translated **18** times:
`translate( 'We need to check your contact information to make sure you can be reached. Please verify your details using the email we sent you, or your domain will stop working. {{learnMoreLink}}Learn more.{{/learnMoreLink}}' )` *ES Score: 7*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).